### PR TITLE
Release/2.1.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
Added

* UI: The cross device feature now supports sending the link via SMS. Users will still be able to copy the link to clipboard.
* UI: Introduced a back button that allows the user to navigate to the previous screen.
* Internal: Introduced code splitting and lazy loading

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-276-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-276-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "2.0.0",
+  "version": "2.1.0-rc.1",
```
Comment (dawiddylowicz) [inline package.json]: just a reminder to change it to 2.1.0 before merging

File: src/components/crossDevice/CrossDeviceLink/style.css [hunk 2]
```diff
@@ -111,7 +110,7 @@
   line-height: 24px;
   color: #2C3E4F;
   white-space: nowrap;
-  font-family: "Open Sans", sans-serif !important;
+  font-family: "Open Sans", sans-serif;
```
Comment (rfreitas) [inline src/components/crossDevice/CrossDeviceLink/style.css]: is this font different from the rest?

Comment (stefaniacardenas) [reply src/components/crossDevice/CrossDeviceLink/style.css]: No, it's not. But for some reason on JSFiddle it was appearing as system-ui, so I had to explicitly add the font family.

Comment (rfreitas) [reply src/components/crossDevice/CrossDeviceLink/style.css]: we should have a tech debt issue to track it then

Comment (stefaniacardenas) [reply src/components/crossDevice/CrossDeviceLink/style.css]: https://jira.onfido.co.uk/browse/CX-1565

**Pull request merged**